### DISCUSSION TITLE
Doku - CentroidCorners

### DISCUSSION THREAD

**Discussion by Arch4060** [pysal/momepy#531] [General]
https://docs.momepy.org/en/stable/generated/momepy.CentroidCorners.html
buildings_df['ccd_means'] = ccd.means 
Should be:
buildings_df['ccd_means'] = ccd.mean

--> ccd.mean[s]

**Comment by jGaboardi** [comment]
@Arch4060 Thanks! Would you be interested in pushing up a fix? If not, no problem, we can fix.

**Reply by Arch4060** [reply]
Have never tried / dones this

**Reply by martinfleis** [reply]
fixed on main. thanks for flagging it up!
